projects/pc/windows/visual_studio/aws_demos/aws_demos.vcxproj.filters

```diff
@@ -1234,6 +1232,9 @@
 		<ClCompile Include="..\..\..\..\..\demos\jobs_for_aws\jobs_demo.c">
 			<Filter>demos\jobs_for_aws</Filter>
 		</ClCompile>
+		<ClCompile Include="..\..\..\..\..\demos\ota\ota_demo_core_http\ota_demo_core_http.c">
```

Comment:
Is this file added intentionally?

Reply:
Yes, the project file were not updated after addition on OTA-HTTP demos